openapi/workday.compensation/Ballerina.toml

```
@@ -0,0 +1,9 @@
+[package]
+license = ["Apache-2.0"]
+keywords = ["Human Resources/HRM", "Cost/Paid"]
+org = "ballerinax"
+name = "workday.compensation"
+distribution = "slbeta3"
+repository = "https://github.com/ballerina-platform/ballerinax-openapi-connectors"
+version = "1.0.0-SNAPSHOT"
```

Shall we remove the `-SNAPSHOT` part from the version?
Refer https://github.com/ballerina-platform/ballerina-extended-library/issues/134
```suggestion
version = "1.0.0"
```

---

+1